src/js/tabs/trade.controller.js

```diff
@@ -615,8 +615,13 @@ TradeTab.prototype.angular = function(module)
       tx.on('success', function(res) {
         setEngineStatus(res, true, type);
 
+
         if (!modifying) order.mode = 'done';
 
+        $timeout(function() {
```

Comment:
@vhpoet Maybe not good to move the buy/sell module (when notification screen goes away). Should we move this notification somewhere so it doesn't move the modules?